atomrdf/structure.py

```diff
@@ -318,6 +336,20 @@ def _make_dislocation(
     positions = np.column_stack(
         (atom_df["pos[0]"].values, atom_df["pos[1]"].values, atom_df["pos[2]"].values)
     )
+
+    #find dislocation character
+    angle = np.dot(dislocation_line, burgers_vector)/(np.linalg.norm(dislocation_line)*np.linalg.norm(burgers_vector))
+    angle_rad = np.arccos(angle)
+    angle_deg = np.degrees(angle_rad)
+
+    disl_dict = {
+        'BurgersVector': burgers_vector,
+        'SlipPlane': slip_plane,
+        'SlipVector': slip_direction,
+        'DislocationLine': dislocation_line,
+        'DislocationCharacter': angle_deg,
```

Comment:
should we also keep the angle?

Reply:
This would actually be useful, especially for mixed dislocations for storing their screw and edge characters. I have raised an issue over at https://github.com/OCDO/ldo/issues/1. We can merge this and open a different one when ontology is updated.